Server/mods/deathmatch/logic/CMainConfig.h

```diff
@@ -84,7 +84,10 @@ class CMainConfig : public CXMLConfig
     bool                        GetJoinFloodProtectionEnabled() { return m_bJoinFloodProtectionEnabled; };
     bool                        GetScriptDebugLogEnabled() { return m_bScriptDebugLogEnabled && !m_strScriptDebugLogFile.empty(); };
     const std::string&          GetScriptDebugLogFile() { return m_strScriptDebugLogFile; };
-    unsigned int                GetScriptDebugLogLevel() { return m_uiScriptDebugLogLevel; };
+
+    const std::set<SString>&    GetScriptDebugLogLevelMap() const noexcept { return m_scriptDebugLogLevelMap; };
+    std::set<SString>&          GetScriptDebugLogLevelMap() noexcept { return m_scriptDebugLogLevelMap; };
```

Comment:
Do we really need the write access to this map from the outside of the class?

Reply:
Maybe. Lets leave it for now and we'll see later